java-client/session-dagger/src/test/java/io/deephaven/client/AnnotatedTables.java

```diff
@@ -0,0 +1,271 @@
+            public <T extends TableOperations<T, T>> T create(TableCreator<T> creation) {
+                final T t = in.logic().create(creation);
+                return creation.merge(t, t, t);
+            }
+        };
+        if (in.isStatic()) {
+            return AnnotatedTable.ofStatic(merge3, in.size() * 3);
+        } else {
+            return AnnotatedTable.ofDynamic(merge3);
+        }
+    };
+
+    public static Map<String, AnnotatedTable> explicitlyAnnotatedTables() {
+        // I wish there were any easy way to get variable names or lambda function names from java :(
+        final Map<String, AnnotatedTable> tables = new LinkedHashMap<>();
+        tables.put("EMPTY_0", EMPTY_0);
+        tables.put("EMPTY_1", EMPTY_1);
+        tables.put("EMPTY_2", EMPTY_2);
+        tables.put("TIME_1", TIME_1);
+        tables.put("TIME_2", TIME_2);
+        tables.put("MERGE_STATIC", MERGE_STATIC);
+        tables.put("MERGE", MERGE);
+        tables.put("VIEW_STATIC", VIEW_STATIC);
+        tables.put("VIEW", VIEW);
+        tables.put("UPDATE_VIEW_STATIC", UPDATE_VIEW_STATIC);
+        tables.put("UPDATE_VIEW", UPDATE_VIEW);
+        tables.put("UPDATE_STATIC", UPDATE_STATIC);
+        tables.put("UPDATE", UPDATE);
+        tables.put("SELECT_STATIC", SELECT_STATIC);
+        tables.put("SELECT", SELECT);
+        tables.put("HEAD_STATIC", HEAD_STATIC);
+        tables.put("HEAD", HEAD);
+        tables.put("TAIL_STATIC", TAIL_STATIC);
+        tables.put("TAIL", TAIL);
+        return tables;
+    }
+
+    public static Map<String, Adapter> adapters() {
+        final Map<String, Adapter> map = new LinkedHashMap<>();
+        map.put("REVERSE", REVERSE_ADAPTER);
+        map.put("HEAD_0", HEAD_0_ADAPTER);
+        map.put("HEAD_1", HEAD_1_ADAPTER);
+        map.put("HEAD_2", HEAD_2_ADAPTER);
+        map.put("TAIL_0", TAIL_0_ADAPTER);
+        map.put("TAIL_1", TAIL_1_ADAPTER);
+        map.put("TAIL_2", TAIL_2_ADAPTER);
+        map.put("BY", BY_ADAPTER);
+        map.put("MERGE_2", MERGE_2);
+        map.put("MERGE_3", MERGE_3);
+        return map;
+    }
+
+    public static Map<String, AnnotatedTable> annotatedTables() {
+        final Map<String, AnnotatedTable> map = explicitlyAnnotatedTables();
+        final Map<String, AnnotatedTable> extras = new LinkedHashMap<>(map);
+        for (Entry<String, AnnotatedTable> e : map.entrySet()) {
+            final String key = e.getKey();
+            final AnnotatedTable value = e.getValue();
+            for (Entry<String, Adapter> a1 : adapters().entrySet()) {
+                extras.put(key + " + " + a1.getKey(), a1.getValue().apply(value));
+            }
+            /*
+             * for (Entry<String, Adapter> a1 : adapters().entrySet()) { for (Entry<String, Adapter> a2 :
+             * adapters().entrySet()) { extras.put(key + " + " + a1.getKey() + " + " + a2.getKey(),
+             * a2.getValue().apply(a1.getValue().apply(value))); } }
+             */
```

Comment:
This commented snippet of code has to be addressed (removed, or integrated -- whichever is appropriate).